app/assets/javascripts/darkswarm/services/map_centre_calculator.js.coffee

```diff
@@ -0,0 +1,36 @@
+Darkswarm.factory 'MapCentreCalculator', (Enterprises, openStreetMapConfig) ->
+  new class MapCentreCalculator
+
+    initialLatitude: =>
+      if Enterprises.geocodedEnterprises().length > 0
+        @_calculate("latitude", Enterprises.geocodedEnterprises())
+      else
+        openStreetMapConfig.open_street_map_default_latitude
+
+    initialLongitude: =>
+      if Enterprises.geocodedEnterprises().length > 0
+        @_calculate("longitude", Enterprises.geocodedEnterprises())
+      else
+        openStreetMapConfig.open_street_map_default_longitude
+
+    _calculate: (angleName, coordinates) =>
+      positiveAngles = []
+      negativeAngles = []
```

Comment:
these two arrays can be removed right? We append values to them but I don't see them used.

Reply:
@sauloperez ah yes, I should have spotted that, it's fixed now in 
e6ab2ae